core/src/main/scala/com/databricks/labs/remorph/parsers/snowflake/SnowflakeRelationBuilder.scala

```diff
@@ -71,12 +72,15 @@ class SnowflakeRelationBuilder
     }
 
   override def visitSelectOptionalClauses(ctx: SelectOptionalClausesContext): ir.LogicalPlan = {
+
```

Comment:
revert

Reply:
Done. Its work in progress and a draft PR, As a practice I commit the dev code on regular basis.